### ISSUE TITLE
Error: RBDL Addon URDFReader not enabled!

### ISSUE THREAD

**Issue by tianbaoloin** [ORB-HD/rbdl-orb#95]
Thanks for your contribution，when i install rbdl，I get an error every time I import the model，The error message is as follows：
**Error: RBDL Addon URDFReader not enabled!**
I did the same test with PINNOCHIO，PINOCCHIO can load models，The running result is as follows：
_import pinocchio as pin
model1 = pin.buildModelFromUrdf(
    '/home/tian/Desktop/Aircontrolenv/Aircontrol/URDFFILE/twoleg/simplify1fourleg1/urdf/simplify1fourleg1nobase.urdf',
    pin.JointModelFreeFlyer())
print('model1 name: ' + model1.name)
print('model1', model1)
import rbdl
aaa = rbdl.loadModel(
    '/home/tian/Desktop/Aircontrolenv/Aircontrol/URDFFILE/twoleg/simplify1fourleg1/urdf/simplify1fourleg1nobase.urdf')_

The result is as follows：
_model1 name: simplify1fourleg1
model1 Nb joints = 14 (nq=19,nv=18)
  Joint 0 universe: parent=0
  Joint 1 root_joint: parent=0
  Joint 2 jointleftbehind1: parent=1
  Joint 3 jointleftbehind2: parent=2
  Joint 4 jointleftbehind3: parent=3
  Joint 5 jointleftfront1: parent=1
  Joint 6 jointleftfront2: parent=5
  Joint 7 jointleftfront3: parent=6
  Joint 8 jointrightbehind1: parent=1
  Joint 9 jointrightbehind2: parent=8
  Joint 10 jointrightbehind3: parent=9
  Joint 11 jointrightfront1: parent=1
  Joint 12 jointrightfront2: parent=11
  Joint 13 jointrightfront3: parent=12

Error loading model /home/tian/Desktop/Aircontrolenv/Aircontrol/URDFFILE/twoleg/simplify1fourleg1/urdf/simplify1fourleg1nobase.urdf!
Error: RBDL Addon URDFReader not enabled!_

There are many commands in rbdl.，as following:
![1650423386722](https://user-images.githubusercontent.com/94944742/164141658-ec3670ab-6635-4060-9613-6f045fba8a22.jpg)

I execute the command and the prompt is as follows：
_(base) tian@tian:~/Documents/rbdl-orb-build$ cmake -D CMAKE_BUILD_TYPE=Release ../rbdl-orb -D RBDL_BUILD_ADDON_URDFREADER=ON 
CMake Error at addons/urdfreader/thirdparty/CMakeLists.txt:3 (MESSAGE):
  The urdfreader module was not initalized correctly! Please make sure that
  you recursively cloned rbdl.  For more information take a closer look at
  the README!

-- Configuring incomplete, errors occurred!
See also "/home/tian/Documents/rbdl-orb-build/CMakeFiles/CMakeOutput.log"._

I don't know what's wrong, can you give me some advice on how to deal with it？，Looking forward to your reply

**Comment by ju6ge** [comment]
Hey, this is a simple fix! The Error Message tells you to read the README … This is the section that is relevant to you:

## Building RBDL from Source

The official rbdl-orb git repository can be cloned from

    https://github.com/ORB-HD/rbdl-orb

(See https://git-scm.com/downloads/guis/[](https://github.com/ORB-HD/rbdl-orb#upgrading-from-an-older-version-of-rbdl) for git clients.)

To make sure all submodules are correctly downloaded, clone the repository recursively!
```
git clone --recurive https://github.com/ORB-HD/rbdl-orb
```
Upgrading from an older version of RBDL

For convenience there is a script to upgrade to the newest RBDL repository version.
```
./upgrade.sh
```
It pulls the latest commits from master and also checks out the correct version of all sub repositories. Manual upgrading requires doing the following:
```
git pull origin master
git submodule update --init
```

Basically, you did not clone the repository recursively. The URDF Parser is included as a git submodule, this has the advantage of it being easier to update the parsers code and keep rbdl in sync. When you don't clone recursively the urdf code will be missing on your machine! Either reclone the entire repo or use the commands listed at the end. To initialize and update the submodules.
